README.md

```diff
@@ -32,19 +32,19 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `npsfile.js` and use `nps` in a single `package.json` script:
 
 **package.json**
 
 ```json
 {
   "scripts": {
-    "start": "package-scripts"
+    "start": "npm-package-scripts"
   }
 }
 ```
 
-**package-scripts.js**
+**npsfile.js**
```

Comment:
:+1: for `package-scripts.config.js`

I'm on the fence about the renaming of the project. Typing `p-s` is obviously more cumbersome than `nps`, so I'd be very much in favor of having it as an alias. However I'm not sure if I would rename the whole project to `nps` -- seeing it causes my brain to immediately process it as `npm`. On top of that, it would also take time to see if the maintainer of the current `nps` is willing to part with the name - plausible since the project seems to have moved from the CLI to http://npmsearch.com/ . If they agree to part with the name, they might still choose to keep their project on the registry. In that case, I actually think the logistics of having two tools inhabit the same 'space' would be too painful to deal with for it to be a good option.

`p-s` doesn't have any of those problems. Sure, it's more difficult to type than `nps`, but the alias can still stay there. Would there still be three aliases?

I guess what I'm getting at is that, if the project is to be renamed, I would probably rename it to something that more clearly states what it is rather than something that, best case scenario, doesn't confuse things any more than they currently are.